update/files/multiFileWriter.go

```diff
@@ -44,6 +44,12 @@ func NewMultiFileWriter(args ArgsNewMultiFileWriter) (*multiFileWriter, error) {
 		exportStore:  args.ExportStore,
 	}
 
+	err := os.Mkdir(m.exportFolder, os.ModePerm)
+	if err != nil {
+		_ = os.Remove(m.exportFolder)
```

Comment:
This returns an error if the folder exists. Suggestion to add a check if the folder exists and try to create only if missing.

Reply:
done.